ios/App/App/PluginSafeAreasColor.swift

```diff
@@ -0,0 +1,78 @@
+import Capacitor
+import UIKit
+
+@objc(SafeAreasColorPlugin)
+public class SafeAreasColorPlugin: CAPPlugin, CAPBridgedPlugin {
+    public let identifier = "SafeAreasColorPlugin"
+    public let jsName = "SafeAreasColor"
+    public let pluginMethods: [CAPPluginMethod] = [
+        CAPPluginMethod(name: "changeSafeAreasColorOniOS", returnType: CAPPluginReturnPromise)
+    ]
+
+    @objc func changeSafeAreasColorOniOS(_ call: CAPPluginCall) {
+        let value = call.getString("color") ?? ""
+        guard let uiColor = UIColor(hex: value) else {
+              call.reject("Invalid color format")
+              return
+          }
+    
+        DispatchQueue.main.async {
+                 if #available(iOS 13.0, *) {
+                     // Find the active window in iOS 13 or later
+                     if let window = UIApplication.shared.connectedScenes
+                         .compactMap({ $0 as? UIWindowScene })
+                         .flatMap({ $0.windows })
+                         .first(where: { $0.isKeyWindow }) {
+                         
+                         window.backgroundColor = uiColor // Set the background color
+                         call.resolve(["value": "Color set successfully"])
+                     } else {
+                         call.reject("No active window found")
+                     }
+                 } else {
+                     // For iOS 12 and earlier
```

Comment:
we don't support 12 and earlier, our version minimum atm is 16 - latest iOS is 18

Reply:
I did this condition based on code made for cut Safe Areas. So remove this condition from there as well?

Reply:
we can keep it if you want, just saying that since we only support iOS 16 and above the code is not being used

Reply:
I think you are correct, if both are dead code, they should be removed to not have unnecessary code. 

And the condition is using 13.0, we can increase this number to 16 as well.

Reply:
Changed